algorithms/Python/linked_lists/reverse-linkedlist.py

```diff
@@ -0,0 +1,51 @@
+# reverse a given linkedlist
+
+class Node:
+  # Constructor to initialize the node object
+    def __init__(self, data=0, next=None):
+        self.data = data
+        self.next = next
+
+
+class LinkedList:
+    # Constructor to initialize head
+    def __init__(self) -> None:
+        self.head = None
+
+     # Function to insert a new node at the beginning
+    def push(self, data):
+        node = Node(data)
+        node.next = self.head
+        self.head = node
+
+     # Function to print the list from begining
```

Comment:
There is a typo which due to which `codespell` check is failing.

```suggestion
     # Function to print the list from beginning
```

Reply:
Fixed